TestStack.Dossier/TestDataBuilder.cs

```diff
@@ -144,9 +155,10 @@ public TValue Get<TValue>(Expression<Func<TObject, TValue>> property)
         /// <returns></returns>
         public object Get(Type type, string propertyName)
         {
-            if (!Has(propertyName))
-                return Any.Get(type, propertyName);
-            return _properties[propertyName];
+            Func<object> factory;
+            if (_properties.TryGetValue(propertyName, out factory)) return factory();
```

Comment:
Sure! Do you have the coding conventions documented somewhere? I looked for a .md file somewhere and didn't see one.